data.md

```diff
@@ -0,0 +1,167 @@
+WASM Data
```

Comment:
"data" is not too indicative, how about types.md / WASM types?

Reply:
Well, the problem is that it declares more than just types. It also declares data-structures (like the stack) and operations over them, as well as Numbers. I'm open to changing the name, but don't think that `types.md` is the correct name.

I think I'm going to move forward with merge until a clearly better name is suggested.